source/guide1.html

```diff
@@ -0,0 +1,62 @@
+---
+title: Elixir Girls Guides Slackir App retrieving data
+---
+
+<%= partial "partials/guides-nav" %>
```

Comment:
If you rebase onto master, and put this into `source/guides/`, you don't need wrapping stuff, it's in a layout.